todo/src/Actions/todoActions.js

```diff
@@ -0,0 +1,9 @@
+import { ADD_TODO  } from './actionTypes'
+
+export const addTodo = (todo) => {
+    return {
+        type: ADD_TODO,
+        payload: todo
+    }
+}
+
```

Comment:
Your toggleActions would be in the same file as this one, theres no need to separate them int different files